scripts/fuzz_opt.py

```diff
@@ -126,10 +152,10 @@ def fix_spec_output(out):
   for i in range(len(before)):
     compare(before[i], after[i], 'comparing between builds at ' + str(i))
   # fuzz binaryen interpreter itself. separate invocation so result is easily fuzzable
-  run(['bin/wasm-opt', 'a.wasm', '--fuzz-exec', '--fuzz-binary'] + opts)
+  run([in_bin('wasm-opt'), 'a.wasm', '--fuzz-exec', '--fuzz-binary'] + opts)
   # check for determinism
-  run(['bin/wasm-opt', 'a.wasm', '-o', 'b.wasm'] + opts)
-  run(['bin/wasm-opt', 'a.wasm', '-o', 'c.wasm'] + opts)
+  run([in_bin('wasm-opt'), 'a.wasm', '-o', 'b.wasm'] + opts)
+  run([in_bin('wasm-opt'), 'a.wasm', '-o', 'c.wasm'] + opts)
   assert open('b.wasm').read() == open('c.wasm').read()
```

Comment:
This isn't part of this PR, but it would be good to raise an exception with a helpful message here for consistency in error reporting.

Reply:
Adding.